languages/swift/exercises/concept/closures/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+In this exercise you will be using closures with a higher-order function to generate sequences of numbers from a seed number.
```

Comment:
Could you perhaps add a little story/theme to the exercise? It doesn't have to be a big story or anything, but just a little something to give the exercise a bit of story/theme.